.github/workflows/local-dev-ci.yaml

```
@@ -0,0 +1,58 @@
+name: Execute local test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+      - "**/*.md"
+
+jobs:
+  local-dev-test:
+    name: local-dev-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: v1.17
+
+      # Install kind with a local registry
+      - uses: container-tools/kind-action@v1
+        name: Kubernetes KinD Cluster w/local registry
+        with:
+          kubectl_version: v1.23.3
+
+      - name: Checkout kcp repository
+        uses: actions/checkout@v3
+        with:
+          repository: kcp-dev/kcp
+          ref: release-0.6
+      - name: Build kcp binaries
+        run: |-
+          make install
+          kubectl-kcp --version
+
+      - name: Checkout pipeline-service repository
+        uses: actions/checkout@v3
+      - name: Set up Kubernetes workload clusters
+        run: |-
+          ./local/kind/setup.sh
+
+      - name: Set up a kcp cluster.
+        run: |-
+          nohup ./local/kcp/start.sh > nohup.out &
+          i=0
+          while ! grep "ctrl-C" nohup.out >/dev/null; do
+            sleep 5
+            i=$((i+1))
+            if [ $i -gt 70 ]; then
+              printf "Failed to set up a kcp cluster\n"
+              cat nohup.out
+              exit 1
+            fi
+          done
+          cat nohup.out
```

```suggestion
          cat nohup.out
          
```
Please add a terminating newline.

---

fixed.